### DISCUSSION TITLE
How to pass argument 'user_multi_procs=True' using Driver(uc=True)?

### DISCUSSION THREAD

**Discussion by ZcodeZ777** [seleniumbase/SeleniumBase#3273] [Q&A] [answered]
I currently have a driver in python that is run with undetected-chromedriver:
`driver = undetected_chromedriver.Chrome(use_subprocess=True, user_multi_procs=True, headless=True)`

But I am trying to change this to a seleniumbase uc driver. How do I pass the argument 'user_multi_procs=True' into a Driver object?

**Comment by mdmintz** [comment, marked as answer]
Use the following code instead to enable safe multithreading in your files:

```python
import sys
sys.argv.append("-n")
```

If multiple files, you can check to see if it's already there before adding it:

```python
import sys
if "-n" not in sys.argv:
    sys.argv.append("-n")
```

**Reply by ZcodeZ777** [reply]
Thanks 👍
